airflow/dags/marts_ndc/dag.py

```diff
@@ -0,0 +1,63 @@
+import pendulum
+
+from airflow_operator import create_dag
+from common_dag_tasks import get_most_recent_dag_run
+from airflow.decorators import dag,task
+from airflow.operators.trigger_dagrun import TriggerDagRunOperator
+from airflow.hooks.subprocess import SubprocessHook
+
+def run_dag_condition(dag_id):
+    last_run = get_most_recent_dag_run(dag_id)
+    # if a DAG from the list of dependencies is more than 5 days stale
+    if last_run is None or (pendulum.now() - last_run.execution_date).days > 5:
+        return True
+    else:
+        print(f"{dag_id} not being run")
+        return False
+
+def get_dag_list():
+    list_of_dags = []
+    dag_dependencies = ["fda_ndc","fda_unfinished","fda_excluded","rxnorm","rxclass_atc_to_product","rxnorm_historical"]
+    for dag in dag_dependencies:
+        if run_dag_condition(dag):
+            list_of_dags.append(dag)
+    print(f'list of dags to run{list_of_dags}')
+    return list_of_dags
+
+dag = create_dag(
+    dag_id="build_marts",
+    schedule = "0 5 * * 2",
```

Comment:
Only die hard know how to read these chron schedule expressions, I would add a comment to the end saying the frequency in english. In this case “At 05:00 on Tuesday.”

Reply:
got it, fixed!